public/js/controllers/ticketpool_controller.js

```diff
@@ -75,14 +74,17 @@ function priceGraphData (items, memP) {
   return p
 }
 
-function getVal (val) { return isNaN(val) ? 0 : val }
-
-function outputsGraphData (items) {
-  return [
-    getVal(items.solo),
-    getVal(items.pooled),
-    getVal(items.txsplit)
-  ]
+function populateOutputs (data) {
+  var totalCount = 0
+  data.count.map((n) => { totalCount += n })
+  var tableData = '<thead><td># of Outputs</td><td>Count</td><td>% Occurence</td></thead>'
+  data.outputs.map((n, i) => {
+    var count = data.count[i]
+    tableData += '<tr><td class="pr-2 lh1rem vam nowrap xs-w117 font-weight-bold">' + n + '</td>' +
+    '<td><span class="hash lh1rem"> ' + count + ' </span></td>' +
+    '<td><span class="hash lh1rem"> ' + ((count * 100) / totalCount).toFixed(4) + '% </span></td></tr>'
+  })
+  document.getElementById('outputs-data').innerHTML = tableData
```

Comment:
For consistency, let's switch up this implementation to use a stimulus `data-target` instead of an `id` please? Either make `populateOutputs` a controller method, or return the HTML from `populateOutputs`, and set it in `processData`.